dev-resources/logback.xml

```diff
@@ -5,11 +5,11 @@
         </encoder>
     </appender>
 
-    <logger name="puppetlabs" level="error"/>
+    <logger name="puppetlabs" level="trace"/>
```

Comment:
i have definitely seen this affect logging tests. when i've hit those i've refactored the tests to test something different. are we actually checking for logevents that only happen at trace?